src/Sdk/Coverage/types.ts

```diff
@@ -0,0 +1,12 @@
+import { Coverage, Pagination } from 'types';
+
+export interface Options {
+    token: string;
+    baseUrl?: string;
```

Comment:
I see that later you use `Required<Options>` in Coverages constructor. 
Why is `baseUrl` optional here? I see that the usage in Coverages constructor is the only one, so we can simplify things.

```suggestion
    baseUrl: string;
```

And then remove `Required<>` from Coverages constructor.

Reply:
It's also used in the `Sdk`, where it should be optional.

Reply:
> It's also used in the Sdk, where it should be optional.

Sdk uses `Options` interface from `src/Sdk/types.ts`, so a different one. 

So my suggestion is still valid.

Reply:
Nope, it imports `import { Options } from '../types';` So the type here is unused in the version of the code when you commented. 

So this one is unnecessarily duplicated. I moved the type to the `Coverage.ts` file, similarly to how we have `Props` next to the component. And I have applied the your suggestion to not use the `Required`.